src/utils/responses.js

```diff
@@ -4,6 +4,12 @@ const messages = {
     success: 'User Created Successfully',
     error: 'Could not sign up user try again',
     conflict: 'User with that email already exist'
+  },
+  signIn: {
+    success: 'Successfully Signed In',
+    notfound: 'User Cannot be found',
+    error: 'Could not sign in user try again',
+    invalid: 'Invalid Password'
```

Comment:
Hi @oyewoas , Both `notfound` and `invalid` keys should point to `Authorization fail` or `Invalid email or password`, just for security concerns

Reply:
ok @Oliver-ke , the reason I did this is so the user can know what is really wrong, the email may be correct and password incorrect, and in another case, the user has never registered on the app...